app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -436,8 +436,18 @@ private void enableDeleteButton(boolean visibility) {
     private void rebuildCatList() {
         categoryContainer.removeAllViews();
         // @fixme add the category items
-        for (String cat : categoryNames) {
-            View catLabel = buildCatLabel(cat, categoryContainer);
+
+        //As per issue #1826, some categories come suffixed with strings prefixed with |. As per the discussion
```

Comment:
Maybe here you can link "the discussion" , so that everyone can understand later

Reply:
Added